wa/commands/create.py

```diff
@@ -77,7 +156,121 @@ class CreateCommand(ComplexCommand):
     object-specific arguments.
     '''
     subcmd_classes = [
-        #CreateWorkloadSubcommand,
-        #CreatePackageSubcommand,
+        CreateWorkloadSubcommand,
         CreateAgendaSubcommand,
+        #CreatePackageSubcommand,
     ]
+
+
+def create_workload(name, kind='basic', where='local', check_name=True, **kwargs):
+    if check_name:
+        if name in [wl.name for wl in pluginloader.list_plugins('workload')]:
+            raise CommandError('Workload with name "{}" already exists.'.format(name))
+
+    class_name = get_class_name(name)
+    if where == 'local':
+        workload_dir = _d(os.path.join(settings.plugins_directory, name))
+    else:
+        workload_dir = _d(os.path.join(where, name))
+
+    if kind == 'basic':
+        create_basic_workload(workload_dir, name, class_name, **kwargs)
+    elif kind == 'apk':
+        create_apk_workload(workload_dir, name, class_name, **kwargs)
+    elif kind == 'revent':
+        create_revent_workload(workload_dir, name, class_name, **kwargs)
+    elif kind == 'apk_revent':
+        create_apk_revent_workload()(workload_dir, name, class_name, **kwargs)
+    elif kind == 'uiauto':
+        create_uiautomator_workload(workload_dir, name, class_name, **kwargs)
+    elif kind == 'apk_uiauto':
+        create_apk_uiautomator_workload(workload_dir, name, class_name, **kwargs)
+    else:
```

Comment:
Could simplify this a little by having a `dict` mapping the kind name onto the corresponding function and then just doing

```
try:
    create_funcs[kind](worload_dir, name, class_name, **kwargs)
except KeyError:
     raise CommandError('Unknown workload type: {}'.format(kind))
```

Reply:
Addressed comments.